packages/client/src/base/feedback/update-model-command.ts

```diff
@@ -59,38 +55,13 @@ export class FeedbackAwareUpdateModelCommand extends UpdateModelCommand {
     }
 
     protected override performUpdate(oldRoot: GModelRoot, newRoot: GModelRoot, context: CommandExecutionContext): CommandReturn {
-        if (this.feedbackActionDispatcher && this.actionHandlerRegistry) {
-            // Create a temporary context which defines the `newRoot` as `root`
-            // This way we do not corrupt the redo/undo behavior of the super class
-            const tempContext: CommandExecutionContext = {
-                ...context,
-                root: newRoot
-            };
-
-            const feedbackCommands = this.getFeedbackCommands(this.actionHandlerRegistry);
-            feedbackCommands.forEach(command => command.execute(tempContext));
-        }
-
+        // Create a temporary context which defines the `newRoot` as `root`
+        // This way we do not corrupt the redo/undo behavior of the super class
+        const tempContext: CommandExecutionContext = { ...context, root: newRoot };
+        this.feedbackActionDispatcher?.applyFeedbackCommands(tempContext);
         return super.performUpdate(oldRoot, newRoot, context);
     }
 
-    protected getFeedbackCommands(registry: ActionHandlerRegistry): Command[] {
```

Comment:
This is a soft API break for adopters that have a customization of the `FeedbackAwareUpdateModelCommand`.
Should be mentioned in the changelog.

Reply:
I do have that written in the PR description, should I also already note it down somewhere in the changelog?